apps/docs/src/app/core/component-docs/dialog/examples/component-based/dialog-example.component.ts

```diff
@@ -15,6 +15,9 @@ import { DIALOG_REF, DialogRef } from '@fundamental-ngx/core';
                         {{ fact }}
                     </li>
                 </ul>
+                <fd-select placeholder="Select an option" [(value)]="selectedValue1">
```

Comment:
Can we do some more consistent/meaningfull example?
![image](https://user-images.githubusercontent.com/17496353/86459324-e5cc4d80-bd26-11ea-8989-47dadc972233.png)